ddtrace/tracer/option.go

```diff
@@ -344,6 +353,23 @@ func newConfig(opts ...StartOption) *config {
 	c.globalSampleRate = sampleRate
 	c.httpClientTimeout = time.Second * 10 // 10 seconds
 
+	c.traceRateLimit = defaultRateLimit
+	origin := telemetry.OriginDefault
+	v := os.Getenv("DD_TRACE_RATE_LIMIT")
+	if v != "" {
```

Comment:
IMHO there is no need for `v` to exist outside of the scope of this test.

```suggestion
	if v := os.Getenv("DD_TRACE_RATE_LIMIT"); v != "" {
```

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/3033/commits/1a458ca46107aeafc2fab5435487ac77d07a5852